### PR TITLE
chore: actually update ui image tag in ops portal

### PR DESCRIPTION
**Pull request by jaxesn**
Like and idiot I forgot we have a image tag override in the values so even tho we updated the kommander chart, it didn't actually update the image

### CODE REVIEW

**Comment by jaxesn** [comment]
I need rebase this on master, but I'm wondering if we should just remove this override now? I believe we had this before because we were dependant on the entire kommander chart and weren't updating the chart regularly. Now that we have a UI chart maybe we just rely on updating the chart if we want to update the image?

File: stable/opsportal/values.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ kommander-ui:
   enabled: true
   image:
     repository: mesosphere/kommander
-    tag: 1.192.0
```
Comment (juliangieseke) [inline stable/opsportal/values.yaml]: wait… that shouldnt be necessary anymore… UI is a subchart now…

**Comment by juliangieseke** [comment]
This is `values.yaml` from https://github.com/mesosphere/kommander/blob/gh-pages/charts/kommander-ui-2.26.1.tgz:
```
image:
  repository: mesosphere/kommander
  defaultTag: "{{ .Chart.AppVersion }}"
  # add this to override default tag: 1.184.0-beta.something
  pullPolicy: IfNotPresent
replicas: 1
logoutRedirectPath: /ops/landing
clusterPollingInterval: 3000
hostPollingInterval: 3000
clusterPollingThrottleFactor: 1
kubernetesVersionsSelection: '["1.16.4", "1.16.3", "1.15.6"]'
kubeaddonsAddress: kommander-kubeaddons-kubeaddons-catalog

# Mode must be either production|konvoy
mode: production

displayName: Kommander Cluster

extraInitContainers:

resources:
  requests:
    memory: "256Mi"
    cpu: "100m"
  limits:
    memory: "512Mi"
    cpu: "500m"

readinessProbe:
  initialDelaySeconds: 15
  periodSeconds: 10
livenessProbe:
  initialDelaySeconds: 15
  periodSeconds: 10
  timeoutSeconds: 3

### This must match the serviceName set in the ingress backend below
service:
  name: kommander

ingress:
  extraAnnotations:
    traefik.ingress.kubernetes.io/priority: "2"
    traefik.ingress.kubernetes.io/redirect-permanent: "true"
    traefik.ingress.kubernetes.io/redirect-regex: "/ops/portal/kommander/ui$"
    traefik.ingress.kubernetes.io/redirect-replacement: "/ops/portal/kommander/ui/"
    traefik.ingress.kubernetes.io/request-modifier: "ReplacePathRegex: ^/ops/portal/kommander/ui/(.*) /$1"
    traefik.ingress.kubernetes.io/auth-type: forward
    traefik.ingress.kubernetes.io/auth-url: http://traefik-forward-auth-kubeaddons.kubeaddons.svc.cluster.local:4181/
    traefik.ingress.kubernetes.io/auth-response-headers: X-Forwarded-User
  path: /ops/portal/kommander/ui
```

**Comment by jaxesn** [comment]
Ill close and reopen this. I'm not sure we can do these ingress changes yet due to a bug in the kommander chart affecting login. I will just remove the tag override for now. Ill verify on a live cluster before reopening

**Pull request closed without merging**